machine.go

```diff
@@ -374,8 +374,10 @@ func (m *Machine) writerKernelModules(w *writerhelper.WriterHelper) error {
 		if mergedUsrSystem() {
 			usrpath = "/usr/lib/modules"
 		}
-		if err := w.CopyFile(path.Join(usrpath, kernelRelease, v)); err != nil {
-			return err
+		if _, err := os.Stat(path.Join(usrpath, kernelRelease, v)); err == nil {
+			w.CopyFile(path.Join(usrpath, kernelRelease, v))
+		} else {
+			w.CopyFile(path.Join(usrpath, kernelRelease, v + ".xz"))
```

Comment:
Removing the error here can cause hard to debug issues if some modules are missing so i'd rather not do that. In general fakemachine should be better at error checking, it being a bit limited was that it started out as a PoC that turned into something actually used. (I've long hoped to get time to refactor the code quite a bit but never got aroudn to it)

Reply:
OK, i reworked that and it's now part of https://github.com/go-debos/fakemachine/pull/45